app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
- [ ] Good job creating association for the blog application, However, as you can notice it in the UML, the foreign key that is used in the `post`, `like` and `comment` table is `author_id`, but you did not clarify it when doing association. You should write

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

```

Reply:
Done as per instructions